From d8538a57553d94290870671db1d5a4fcd4d7e709 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Pablo=20Su=C3=A1rez=20Hern=C3=A1ndez?=
 <psuarezhernandez@suse.com>
Date: Fri, 26 Feb 2021 09:15:03 +0000
Subject: [PATCH] Fix regression on cmd.run when passing tuples as cmd
 (bsc#1182740)

(cherry picked from commit 9a76246adedb60e24a75682077654a352a965cb9)
---
 salt/modules/cmdmod.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/salt/modules/cmdmod.py b/salt/modules/cmdmod.py
index bbc303c3f8..f24e7cc9ae 100644
--- a/salt/modules/cmdmod.py
+++ b/salt/modules/cmdmod.py
@@ -78,7 +78,7 @@ def __virtual__():
 
 
 def _log_cmd(cmd):
-    if not isinstance(cmd, list):
+    if isinstance(cmd, str):
         return cmd.split()[0].strip()
     return cmd[0].strip()
 
-- 
2.30.1


